authorizerd.go

```diff
@@ -171,13 +173,21 @@ func New(opts ...Option) (Authorizerd, error) {
 	if prov.roleProcessor, err = role.New(
 		role.WithPubkeyProvider(pubkeyProvider),
 		role.WithJWKProvider(jwkProvider),
-		role.WithEnableMTLSCertificateBoundAccessToken(prov.atpParams[0].verifyCertThumbprint),
-		role.WithClientCertificateGoBackSeconds(prov.atpParams[0].certBackdateDur),
-		role.WithClientCertificateOffsetSeconds(prov.atpParams[0].certOffsetDur),
 	); err != nil {
 		return nil, errors.Wrap(err, "error create role processor")
 	}
 
+	if prov.accessProcessor, err = access.New(
```

Comment:
seems it is better to check `enable` at here, and check `prov.accessProcessor == nil` inside `initVerifiers()` 

same for other processors

Reply:
agree.

Reply:
c89cb16